### PR TITLE
add migrated episodes to data source and add filters to reports

### PR DESCRIPTION
**Pull request by nickpell**
screen shot 2017-02-20 at 2 03 10 pm

@czue cc: @sravfeyn

### CODE REVIEW

File: custom/enikshay/reports/generic.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from collections import namedtuple
 
-from sqlagg.filters import IN, AND, GTE, LT, RawFilter
+from sqlagg.filters import IN, AND, GTE, LT, RawFilter, EQ
```
Comment (dimagimon) [inline custom/enikshay/reports/generic.py]: F401 'RawFilter' imported but unused

File: custom/enikshay/reports/generic.py [hunk 2]
```diff
@@ -9,12 +9,12 @@
 from corehq.apps.reports.util import get_INFilter_bindparams
 from corehq.apps.userreports.util import get_table_name
 from corehq.sql_db.connections import UCR_ENGINE_ID
-from custom.enikshay.reports.filters import EnikshayLocationFilter, QuarterFilter
+from custom.enikshay.reports.filters import EnikshayLocationFilter, QuarterFilter, EnikshayMigrationFilter
```
Comment (dimagimon) [inline custom/enikshay/reports/generic.py]: F401 'QuarterFilter' imported but unused

File: custom/enikshay/reports/generic.py [hunk 3]
```diff
@@ -9,12 +9,12 @@
...
 from custom.utils.utils import clean_IN_filter_value
 
 TABLE_ID = 'episode'
 
-EnikshayReportConfig = namedtuple('ReportConfig', ['domain', 'locations_id', 'start_date', 'end_date'])
+EnikshayReportConfig = namedtuple('ReportConfig', ['domain', 'locations_id', 'is_migrated', 'start_date', 'end_date'])
```
Comment (dimagimon) [inline custom/enikshay/reports/generic.py]: E501 line too long (118 > 115 characters)

File: custom/enikshay/reports/generic.py [hunk 4]
```diff
@@ -9,12 +9,12 @@
...
 from corehq.apps.userreports.util import get_table_name
 from corehq.sql_db.connections import UCR_ENGINE_ID
-from custom.enikshay.reports.filters import EnikshayLocationFilter, QuarterFilter
+from custom.enikshay.reports.filters import EnikshayLocationFilter, EnikshayMigrationFilter
 from custom.utils.utils import clean_IN_filter_value
 
 TABLE_ID = 'episode'
 
-EnikshayReportConfig = namedtuple('ReportConfig', ['domain', 'locations_id', 'start_date', 'end_date'])
+EnikshayReportConfig = namedtuple('ReportConfig', ['domain', 'locations_id', 'is_migrated', 'start_date', 'end_date'])
```
Comment (dimagimon) [inline custom/enikshay/reports/generic.py]: E501 line too long (118 > 115 characters)

**Review by sravfeyn — APPROVED** [review]

**Review by czue — APPROVED** [review]
sweet thanks! @kkrampa do you want to take a quick look and confirm this looks ok to you?

**Comment by nickpell** [comment]
going to self-merge.  @kkrampa please let me know if you have feedback.

**Pull request merged**